platforms/corda-ent/configuration/roles/helm_component/templates/idman.tpl

```diff
@@ -15,28 +15,20 @@ spec:
     nodeName: {{ node_name }}
     metadata:
       namespace: {{ component_ns }}
-    storage:
-      name: {{ storageclass }}
-    replicas: 1
     image:
       initContainerName: {{ init_container_name }}
-      imagepullsecret: {{ image_pull_secret }}
-    dockerImage:
-      name: {{ idman_image_name }}
-      tag: {{ idman_image_tag }}
+      idmanContainerName: {{ idman_image_name }}:{{ idman_image_tag }}
       pullPolicy: Always
       imagePullSecret: {{ image_pull_secret }}
+    storage:
```

Comment:
need update after comments incorporated for values

Reply:
Fixed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5); changes to `value.yaml` file have been worked into `idman.tpl`